feg/gateway/policydb/serializers_test.go

```diff
@@ -42,11 +42,8 @@ func TestPolicyDeserializer(t *testing.T) {
 	deserializedPolicy, ok := iPolicy.(*lteProtos.PolicyRule)
 	assert.True(t, ok)
 
-	// clear out meta fields
-	clearOutMetaFieldsFromPolicy(policy)
-	clearOutMetaFieldsFromPolicy(deserializedPolicy)
-
-	assert.Equal(t, policy, deserializedPolicy)
+	equal := proto.Equal(policy, deserializedPolicy)
+	assert.True(t, equal)
```

Comment:
Nit: consider combining this pattern into single line

Reply:
Will use the function mentioned in [comment](https://github.com/magma/magma/pull/12151#discussion_r856188709) below instead if that is fine with you.